README.md

```diff
@@ -284,6 +285,7 @@ There are some config settings you need to change in the files below.
 | `allowAnonymous` | `true` or `false` | set to allow anonymous usage (default is `true`) |
 | `allowAnonymousEdits` | `true` or `false` | if `allowAnonymous` is `true`: allow users to select `freely` permission, allowing guests to edit existing notes (default is `false`) |
 | `allowFreeURL` | `true` or `false` | set to allow new note creation by accessing a nonexistent note URL |
+| `forbiddenNoteIDs` | `true` or `false` | disallow creation of notes, even if `allowFreeUrl` is `true` |
```

Comment:
different from the env var description (see above). Reading the code, I believe this one to be faulty.

Reply:
Yes, that's obviously a copy-paste error.

Reply:
Updated.